README.rst

```diff
@@ -165,7 +161,15 @@ THIS IS SOFTWARE IN DEVELOPMENT, DECIDE YOURSELF WHETHER IT FITS YOUR NEEDS.
 Borg is distributed under a 3-clause BSD license, see `License`_
 for the complete license.
 
-|build| |coverage|
+|doc| |build| |coverage|
+
+.. |doc| image:: https://readthedocs.org/projects/borgbackup/badge/?version=stable
+        :alt: Documentation
+        :target: http://borgbackup.readthedocs.org/en/stable/
+
+.. |build| image:: https://travis-ci.org/borgbackup/borg.svg
+        :alt: Build Status
+        :target: https://travis-ci.org/borgbackup/borg
 
 .. |build| image:: https://travis-ci.org/borgbackup/borg.svg
         :alt: Build Status
```

Comment:
it is not duplicate because with `|build|` you reference to something named build. and with `.. |build| <text>` it is declared or I am missing something.